src/vcbuild/msvc-dmc.d

```diff
@@ -19,6 +21,15 @@ int main(string[] args)
     newArgs ~= `/Ivcbuild`;
     newArgs ~= `/Iddmd\root`;
     newArgs ~= `/FIwarnings.h`;
+
+    enum vs2015 = "Microsoft Visual Studio 14.0";
+    if (environment.get("VCINSTALLDIR", "").canFind(vs2015) || environment.get("MSVC_CC", "").canFind(vs2015))

```

Comment:
The check can fail if the installation path is not the default. Checking the version directly using VisualStudioVersion (also set via vcvarsall.bat) might be less brittle.

Reply:
Just pushed changes to do that. Thanks for the comment, I didn't know about `%VisualStudioVersion%`.